AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -112,6 +112,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+karriere.rewe-group.com##.ukCookiePolicy
```

Comment:
@spiri-leo not reproduced on my end. Is it still relevant?

Reply:
It doesn't reproduced for me either.